sms_twilio.js

```diff
@@ -14,6 +14,38 @@ sms.configure = function (options) {
   sms.from = options.from;
 };
 
+var lookups = new Meteor.Collection('meteor_accounts_sms_lookups');
+
+/**
+ * Lookup phone number information through twilio lookups api.
+ * @param phone
+ */
+sms.lookup = function (phone) {
+  var lookup = lookups.findOne({ phone_number: { $regex: phone }});
```

Comment:
ok, accounts package api requires the use of `+1` maybe we should do the same here? I'm thinking the correct nr. is with the land code. Let me know what you think